src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/relatedPartyTransactions/AC7803.scala

```diff
@@ -30,8 +30,8 @@ case class AC7803(value: Option[String]) extends CtBoxIdentifier(name = "Name of
 
     collectErrors(
       validateAsMandatory(this),
-      validateOptionalStringByLength("AC7803", this, 1, StandardCohoTextFieldLimit),
-      validateOptionalStringByRegex("AC7803", this, ValidCoHoCharacters)
+      validateStringMaxLength("AC7803", value.getOrElse(""), StandardCohoTextFieldLimit),
```

Comment:
```
value.getOrElse("")
```
isn't as nice as the previous version. 

Can't we have a function that takes an option the ignores if empty?